### ISSUE TITLE
[Bug] Notification channel 'DefaultChannel' has importance 'Default' - But it is set to high in config

### ISSUE THREAD

**Issue by NGumby** [thomasgalliker/Plugin.FirebasePushNotifications#89]
#### Description
I get this log when receiving a notification and the app is killed - 
2024-11-04 14:15:23.8287|Info|CSNativeApp.App|MainPage ctor
[EOL]
2024-11-04 14:15:24.1098|Info|CSNativeApp.App|WebViewService.Initialize
[EOL]
2024-11-04 14:15:24.4180|Debug|Plugin.FirebasePushNotifications.Platforms.FirebasePushNotificationManager|ConfigurePlatform
[EOL]
2024-11-04 14:15:24.4233|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|SetNotificationChannelGroups: notificationChannelGroupRequests=[]
[EOL]
2024-11-04 14:15:24.4295|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|DeleteNotificationChannelGroups: groupIds=[]
[EOL]
2024-11-04 14:15:24.4295|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|CreateNotificationChannelGroups: notificationChannelGroupRequests=[]
[EOL]
2024-11-04 14:15:24.4354|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|SetNotificationChannels: notificationChannelRequests=[DefaultChannel]
[EOL]
2024-11-04 14:15:24.4449|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|DeleteNotificationChannels: channelIds=[]
[EOL]
2024-11-04 14:15:24.4511|Debug|Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannels|CreateNotificationChannels: notificationChannelRequests=[DefaultChannel]
[EOL]
2024-11-04 14:15:24.5004|Debug|Plugin.FirebasePushNotifications.Platforms.PNFirebaseMessagingService|HandleIntent: Action=com.google.android.c2dm.intent.RECEIVE
[EOL]
2024-11-04 14:15:24.5294|Debug|Plugin.FirebasePushNotifications.Platforms.FirebasePushNotificationManager|HandleNotificationReceived
[EOL]
2024-11-04 14:15:24.5465|Debug|Plugin.FirebasePushNotifications.Platforms.FirebasePushNotificationManager|HandleNotificationReceived queues event "NotificationReceived" into PersistentQueue<FirebasePushNotificationDataEventArgs> for deferred delivery
[EOL]
2024-11-04 14:15:24.5547|Debug|Plugin.FirebasePushNotifications.Model.Queues.PersistentQueue|WriteQueueFile: fileInfo=/data/user/0/ca.compusport.compusport/files/Documents/FirebaseQueues/notificationReceivedQueue.json
[EOL]
2024-11-04 14:15:25.0419|Debug|CSNativeApp.App|OnReceived AcknowledgeNotification 31446792
[EOL]
2024-11-04 14:15:25.2902|Debug|Plugin.FirebasePushNotifications.Platforms.NotificationBuilder|OnNotificationReceived
[EOL]
2024-11-04 14:15:25.3244|Warn|Plugin.FirebasePushNotifications.Platforms.NotificationBuilder|Notification channel 'DefaultChannel' has importance 'Default' which is lower than notification importance 'High'

But it is set to high in UseFirebasePushNotifications

#### Steps to Reproduce

Here's my code:
                o.Android.DefaultNotificationChannelImportance = Android.App.NotificationImportance.High;
                o.Android.UseBigTextStyle = false;
                o.Android.DefaultColor = SharedSettings.NotificationColor.ToAndroid();
                o.Android.NotificationActivityType = typeof(MainActivity);
                var channel =  
                    new Plugin.FirebasePushNotifications.Platforms.Channels.NotificationChannelRequest{
                        ChannelId = "DefaultChannel",
                        ChannelName = "General",
                        Description = "The default notification channel",
                        LockscreenVisibility = Android.App.NotificationVisibility.Public,
                        Importance = Android.App.NotificationImportance.High,
                        LightColor = Android.Graphics.Color.Orange,
                        VibrationPattern = [0,100,150,150,150,200]
                        };
                o.Android.NotificationChannels = new [] { channel };
                o.Android.DefaultNotificationChannelId = channel.ChannelId;

#### Expected Behavior

To not see this log

#### Actual Behavior
- 

#### Basic Information
- Version with issue: 3.1.6-pre
- Last known good version: unknown

**Comment by thomasgalliker** [comment]
What was the content of the notification you sent? I tried to reproduce but the warning message was not logged.

**Comment by thomasgalliker** [comment]
If you uninstall the app and reinstall it, does the warning still appear? If no, it would mean that something is wrong with updating existing notification channels.

**Comment by thomasgalliker** [comment]
I tried to reproduce this issue with following notification message and a default notification channel with Importance=High (just like the one you created). In order to receive this message, I had to subscribe to topic "general" but that shouldn't cause any problems.

```
{
    "message": {
        "topic": "general",
        "notification": {
            "title": "Medication Intake",
            "body": "Do you want to take your medicine?"
        },
        "data": {
            "priority": "high"
        },
        "android": {
            "notification": {
                "click_action": "medication_intake"
            }
        },
        "apns": {
            "payload": {
                "aps": {
                    "category": "medication_intake"
                }
            }
        }
    }
}
```

I stepped trough the code when I received the message and the warning _Notification channel 'DefaultChannel' has importance 'Default' which is lower than notification importance 'High'_  did not appear.

If you can help me to reproduce this issue I'll come back to it.

**Comment by NGumby** [comment]
I should get back to this issue in a few days, got something else I need to finish first. I will try to get a repro. Sorry about that.

________________________________

**Comment by thomasgalliker** [comment]
Don‘t worry 👍🏻 Good to hear I‘m not the only one with a tight schedule 🤓

**Comment by thomasgalliker** [comment]
Happy new year @NGumby 
Could you get back to this already?
